test/animals.js

```diff
@@ -1,5 +1,5 @@
 var expect = require('chai').expect;
-var ObjectTemplate = require('../index.js');
+var ObjectTemplate = require('../dist/index.js').default;
```

Comment:
should we take this time to convert all tests to typescript as well?

Reply:
We have some tests in typescript, I added them to circle ci config (they all pass).  Let's keep these tests in js. I don't want to convert them at the moment either, that's not in the scope of this mr

Reply:
ok sounds good